cypress/page-objects/tenth_challenge.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
+export class tenthPage {
+
+    validateChallengeFifthHeading() {
+        cy.get('section > h2')
+        .should('include.text','Challenge #10: Fetch the password from the "Network" tab')
+    }
+
+    getPasswordFromNetworkRequest() {
+        cy.request('https://hacker-challenge.netlify.app//challenges/normal/jellyfish-password.json').then(
```

Comment:
general note: I think you should run the tests against a localhost hosted version of the hacker challenge to be less dependent and increase speed